frontend/javascripts/test/libs/estimate_affine.spec.ts

```diff
@@ -0,0 +1,32 @@
+import "test/mocks/lz4";
+import _ from "lodash";
+import test from "ava";
+import { Vector3 } from "oxalis/constants";
+import Matrix from "ml-matrix";
+import estimateAffine from "libs/estimate_affine";
+import { almostEqual, getPointsC555 } from "./transform_spec_helpers";
+
+test("Estimate affine projection", (t) => {
+  const [sourcePoints, targetPoints] = getPointsC555();
+  let affineMatrix = estimateAffine(sourcePoints, targetPoints);
+
+  const transform = (x: number, y: number, z: number) => {
+    const vec = new Matrix([[x, y, z, 1]]).transpose();
+    const transformed = affineMatrix.mmul(vec);
+    return transformed.to1DArray() as Vector3;
+  };
+
+  almostEqual(
+    t,
+    [568.1202797015036, 528.013612246682, 1622.1124501555569],
```

Comment:
How did you obtain these ground-truth values you are comparing to?

Reply:
These stem from my reference implementation in python. The input coordinates here are also part of the correspondences, which is why I could sanity-check them easily by comparing to them the target coordinates.